sql/src/planner.rs

```diff
@@ -1289,7 +1289,7 @@ mod tests {
 
     #[test]
     fn test_show_tables_statement_to_plan() {
-        let sql = "SHOW TABLES;";
+        let sql = "show tables;";
```

Comment:
Seems no need to change this?

Reply:
this is to restart the workflow.🤣That bug caused me to modify it back and forth many times